packages/react-checkbox/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@fluentui/react-checkbox",
-  "version": "1.0.0-beta.60",
-  "description": "Fluent UI React Checkbox component",
+  "version": "9.0.0-alpha.0",
+  "description": "React components for building web experiences",
```

Something closer to the previous description is probably better (we ought to update this in the template too)
```suggestion
  "description": "Fluent UI checkbox component",
```